dspace-api/src/main/java/org/dspace/content/dao/impl/BitstreamFormatDAOImpl.java

```diff
@@ -70,21 +75,20 @@ public BitstreamFormat findByMIMEType(Context context, String mimeType, boolean
     @Override
     public BitstreamFormat findByShortDescription(Context context,
                                                   String desc) throws SQLException {
-        Criteria criteria = createCriteria(context, BitstreamFormat.class);
-        criteria.add(Restrictions.and(
-            Restrictions.eq("shortDescription", desc)
-        ));
-
-        return uniqueResult(criteria);
+        CriteriaBuilder criteriaBuilder = getCriteriaBuilder(context);
+        CriteriaQuery criteriaQuery = getCriteriaQuery(criteriaBuilder, BitstreamFormat.class);
+        Root<BitstreamFormat> bitstreamFormatRoot = criteriaQuery.from(BitstreamFormat.class);
+        criteriaQuery.select(bitstreamFormatRoot);
+        criteriaQuery.where(criteriaBuilder.equal(bitstreamFormatRoot.get(BitstreamFormat_.shortDescription), desc));
+        return uniqueResult(context, criteriaQuery, false, BitstreamFormat.class, -1, -1);
     }
 
     @Override
-    public int updateRemovedBitstreamFormat(Context context, BitstreamFormat deletedBitstreamFormat,
-                                            BitstreamFormat newBitstreamFormat) throws SQLException {
+    public int updateRemovedBitstreamFormat(Context context, BitstreamFormat deletedBitstreamFormat, BitstreamFormat
+        newBitstreamFormat) throws SQLException {
```

Comment:
Again, I'm not sure why this code is rewrapping things that were already obeying code style?  It looks like your IDE settings have done very odd wrapping throughout this PR.  There's really no reason to replace line wraps that obey code style already.